ding/envs/env_manager/envpool_env_manager.py

```diff
@@ -55,18 +72,29 @@ def launch(self) -> None:
             seed = 0
         else:
             seed = self._seed
+
+        kwargs = {}
+        if "episodic_life" in self._cfg:
```

Comment:
add default value in default config, so we can omit the if expression and directly conduct the assignment operation

Reply:
fixed.